Sources/LocalReceiptParsing/ReceiptParser-only-files/PurchasesReceiptParser+Fetching.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  ReceiptFetcher.swift
+//
+//  Created by Nacho Soto on 1/10/23.
+
+import Foundation
+
+public extension PurchasesReceiptParser {
+
+    // Note: this is a simplified version of `ReceiptFetcher`
+    // available for public use.
+
+    /// Fetches and parses the local receipt
+    /// - Returns: ``AppleReceipt`` of the parsed local receipt.
+    /// - Throws: ``PurchasesReceiptParser/Error`` if fetching or parsing failed.
+    ///
+    /// - Note: this method won't use ``SKReceiptRefreshRequest`` to
+    ///  fetch the receipt if it's not already available.
```

Comment:
Well I guess if missing this can cause issues in the functionality of the ReceiptParser library, it's possible we will get some questions from users using this... But I think it's ok to go with this for now and deal with it in the future if we are asked about this a lot.

Reply:
yeah, bummer that we don't the ability to refresh for now, but we can figure it out if we do get actual requests for it

Reply:
Agreed!